activate.sh

```diff
@@ -1,7 +1,14 @@
 AUTOENV_AUTH_FILE="${AUTOENV_AUTH_FILE:-$HOME/.autoenv_authorized}"
 AUTOENV_ENV_FILENAME="${AUTOENV_ENV_FILENAME:-.env}"
+AUTOENV_ENV_LEAVE_FILENAME="${AUTOENV_ENV_LEAVE_FILENAME:-.env.leave}"
+# AUTOENV_ENABLE_LEAVE
 
 autoenv_init() {
+
+	if [ ! -z $AUTOENV_ENABLE_LEAVE ]; then
```

Comment:
- `! -z` should be `-n` for simple.
- `$AUTOENV_ENABLE_LEAVE` must be double quotes `"$AUTOENV_ENABLE_LEAVE"`. Variables should always be quoted in shell scripts, except you want `split+glob`.